src/teleop/download_map/download_map.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+import zipfile
+import json
+import boto3
+import subprocess
+
+
+# If System has issues with number of file watcher when using GUI with downloaded map
+# Run "sudo sysctl -w fs.inotify.max_user_watches=100000"
+def main():
+    result = subprocess.Popen(
+        ["rospack", "find", "mrover"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )  # runs rospack find mrover to get directory
+    m, err = result.communicate()
+    m = m.decode("utf-8")  # convert byte to string type
```

Comment:
it'll still complain since it randomly changed from byte to string.
he'll prob want to do
m_bytes, err = result.communicate()
    m = m_bytes.decode("utf-8")  # convert byte to string type